x/fungible/keeper/evm.go

```diff
@@ -756,6 +755,8 @@ func (k Keeper) CallEVMWithData(
 		}
 
 		if !noEthereumTxEvent {
+			attrs = append(attrs, sdk.NewAttribute("TxData", hexutil.Encode(msg.Data()))) // adding txData for more info in rpc methods
+			attrs = append(attrs, sdk.NewAttribute("TxNonce", fmt.Sprint(nonce)))         // adding nonce for more info in rpc methods
```

Comment:
Comments should be above the line. And I think we can make these more descriptive